packages/auth-server/components/session/Metadata.vue

```diff
@@ -27,6 +27,9 @@
     >
       {{ domain }}
     </p>
+    <div v-if="appMeta.configData && appMeta.configData && appMeta.configData['message']">
```

Comment:
nit: This check is being done twice
```suggestion
    <div v-if="appMeta.configData && appMeta.configData['message']">
```